webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/snippet/node/AcceptNode.scala

```diff
@@ -138,9 +138,9 @@ class AcceptNode extends Loggable {
       accept match {
         case f:     Failure  =>
           S.error(
-            <span>
-            {f.messageChain}
-          </span>
+            <div class="alert alert-danger">
```

Comment:
The only thing missing is a warning icon : `<i class ="fa fa-warning"></i>`, and that's it for me :)